src/Futhark/CLI/REPL.hs

```diff
@@ -3,6 +3,7 @@
 -- | @futhark repl@
 module Futhark.CLI.REPL (main) where
 
+import Control.Arrow (ArrowChoice (left))
```

Comment:
I think you can use `Data.Bifunctor.first` instead of this (and that is the common style in the compiler).

Reply:
And done